doc/faq.rst

```diff
@@ -244,3 +243,24 @@ program: Insert the following instructions in your main script::
 
 You can find more default on the new start methods in the `multiprocessing
 documentation <https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods>`_.
+
+How do I set a random_state for an entire execution?
+----------------------------------------------------
+
+For testing and replicability, it is often important to have the entire execution
+controlled by a single seed for the pseudo-random number generator used in algorithms
+that have a randomized component. Scikit-learn does not use its own global random state;
+whenever a RandomState instance or an integer random seed is not provided as an argument,
+it relies on the numpy global random state, which can be set using ''numpy.random.seed'' function.
```

Comment:
@GaelVaroquaux I'm not sure if I understand your notation with ":func:``numpy.random.seed``" I tried inserting the raw input for your above text, just the text (:func:numpy.random.seed), removing the double quotations, and moving the apostrophes outside of the double quotes -- each did not reveal any special formatting. Which should be used?